app/controllers/organizations_controller.rb

```diff
@@ -16,6 +18,45 @@ def new
     @organization = Organization.new
   end
 
+  def new_account
```

Comment:
@dissident it should not be in organization controller. Create separate controller(s) for wizard